File: src/main/java/fr/lewon/web/bot/entities/input/girl/ActionGirlAllSalaries.java
package fr.lewon.web.bot.entities.input.girl;

import fr.lewon.bot.http.body.urlencoded.FUEMember;

public class ActionGirlAllSalaries extends ActionGirl {

	@FUEMember
	private String where = "home";

	@FUEMember
	private boolean mc = true;

	public ActionGirlAllSalaries() {
		super("get_all_salaries");
	}

}

File: src/main/kotlin/fr/lewon/bot/hh/entities/input/girl/ActionGirlAllSalaries.kt
package fr.lewon.bot.hh.entities.input.girl

import com.fasterxml.jackson.annotation.JsonProperty

class ActionGirlAllSalaries : ActionGirl("get_all_salaries") {
    @JsonProperty
    val where = "home"
    @JsonProperty
    val mc = true
}